src/utils/whitelistApis.ts

```diff
@@ -4096,6 +4096,13 @@ export const API_LIST = {
                                 ROLE.PUBLIC,
                                ],
         },
+        '/proxies/v8/data/v2/system/settings/get/cadreConfig': {
```

Comment:
It should be -> '/proxies/v8/data/v2/system/settings/get/:id'

Reply:
made changes according to the review